src/lib/config.ts

```diff
@@ -2,6 +2,10 @@ const NETWORK = import.meta.env.VITE_NETWORK as "mainnet" | "mumbai";
 const DONATION_ADDRESS = import.meta.env.VITE_DONATION_ADDRESS;
 
 const POLYGON_CHAIN_ID = NETWORK === "mainnet" ? 137 : 80001;
+const SFL_ADDRESS =
+  NETWORK === "mainnet"
+    ? "0xD1f9c58e33933a993A3891F8acFe05a68E1afC05"
```

Comment:
@tiffanydys 

Is this the same as `TOKEN_CONTRACT`?

Could we use the `TOKEN_CONTRACT` variable which injects either the testnet or mainnet address for the SFL token?

Reply:
Whoops sorry, didn't see that. Edited it out and made sure it still works:

<img width="1784" alt="Screen Shot 2022-05-30 at 8 09 53 AM" src="https://user-images.githubusercontent.com/18549210/170897158-9d0d6558-b8ed-42d4-9cb1-1e7c1411458d.png">